src/core/Akka.Tests/Pattern/BackoffSupervisorSpec.cs

```diff
@@ -345,22 +361,22 @@ internal class DelayTable : IEnumerable<object[]>
             IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
         }
 
-        [Fact(Skip = "Racy on Azure DevOps")]
+        [Fact]
         public void BackoffSupervisor_must_stop_restarting_the_child_after_reaching_maxNrOfRetries_limit_using_BackOff_OnStop()
         {
-            var supervisor = Create(OnStopOptions(maxNrOfRetries: 2));
+            var supervisor = Create(OnStopOptions(maxNrOfRetries: 2).WithManualReset());
```

Comment:
Supervisor props need to be forced to use manual reset to make sure that it doesn't auto reset the internal restart count mid-spec and fails the spec.